src/config/ConfigHelper.ts

```diff
@@ -339,7 +339,9 @@ export class ConfigHelper {
       }
     }
 
-    config.accessListFactory = contractAddressesConfig.accessListFactory
```

Comment:
Why this change? It makes no difference, except that now we have more lines of code :-)

Reply:
I saw some errors when testing vscode extension and importing the latest version of the lib, that accessListFactory key is undefined from contractAddressesConfig, because it was mandatory and did not have the check before.

Reply:
but `accessListFactory` is not mandatory, this is how it is defined on Config class:
`accessListFactory?: string`
And the changes does nothing